SurveyApp/Extensions/UIKit/UIViewController/UIViewController+Lottie.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  UIViewController+Lottie.swift
+//  SurveyApp
+//
+//  Created by Chananchida F. on 7/11/21.
+//
+
+import UIKit
+
+protocol LottieShowable {
+    
+    func showCompleteSurveyView(completion: EmptyCompletion?)
+}
```

Comment:
I don't think we need to create a common protocol for these Lottie animations, IMO 🤔 
Just simply use it only for the Complete Survey screen, what do you think?

Reply:
@markgravity Do you mean we can simply move show logic to `CompleteSurveyView` like this:
```Swift
extension CompleteSurveyView {
    
    func show(completion: EmptyCompletion? = nil) {
        presentViewOnTopWindow(for: 0.8) {
            completion?()
        }
    }
}
```

and in `SurveyQuestionPresenter` we can call it like this:
```Swift
func didSubmitSurvey(questionCount: Int) {
    ...
    let completeSurveyView = CompleteSurveyView()
    completeSurveyView.show {
        self.view?.popToRootViewController()
    }
}
```